metadata.json

```diff
@@ -38,7 +38,7 @@
     },
     {
       "name": "puppetlabs/concat",
-      "version_requirement": ">= 1.2.5 < 3.0.0"
+      "version_requirement": ">= 1.2.5 <= 4.1.1"
```

Comment:
Usually we only limit on majors. All other modules are also ```< 5.0.0``` since we trust them to use semver.